test/Riok.Mapperly.Tests/Mapping/ReferenceExternalMappingsTests.cs

```diff
@@ -0,0 +1,228 @@
+﻿namespace Riok.Mapperly.Tests.Mapping;
+
+public class ReferenceExternalMappingsTests
+{
+    [Fact]
+    public Task MapValueUseOnStaticSupportsExternalMappings()
+    {
+        var source = TestSourceBuilder.MapperWithBodyAndTypes(
+            """
+            [MapValue("Value", Use = nameof(OtherMapper.NewValue))]
```

Comment:
IMO these should use fullnameof with the `@` prefix. Otherwise we override the c# behaviour of nameof....

Reply:
Changed the behavior and tests according to the feedback previously.